controllers/ArticleController.js

```diff
@@ -450,6 +450,68 @@ class ArticleController {
     opn(`mailto:?subject=${article.title}&body=${process.env.FRONTEND_URL}/article/${slug}`);
     return res.status(200).json({ status: 200, message: 'Sharing article via Email' });
   }
+
+  /**
+   * @author Chris
+   * @param {Object} req
+   * @param {Object} res
+   * @param {*} next
+   * @returns {Object} Returns the response
+   */
+  static async reportArticle(req, res) {
+    const userId = req.currentUser.id;
+    const { slug } = req.params;
+    const { report } = req.body;
+
+    const article = await Article.findOne({ where: { slug } });
```

Comment:
@filschristian here you need also to check if the article is published

Reply:
Ahh thanks, Mr. Caleb. I will definitely check that